### PR TITLE
Add image resize support in CKEditor

### PR DESCRIPTION
**Pull request by dtdesign**
n/t

### CODE REVIEW

File: wcfsetup/install/files/lib/system/html/input/node/HtmlInputNodeImg.class.php [hunk 1]
```diff
@@ -255,4 +264,20 @@ protected function handleSmiley(\DOMElement $element)
...
+    protected function mirrorWidthAttribute(\DOMElement $element): void
+    {
+        $width = $element->getAttribute("data-width");
+        if ($width && $width !== "100%") {
+            $style = $element->getAttribute("style");
+            if ($style !== "") {
+                $style .= "; ";
+            }
+
+            $style .= "widht: {$width}";
```
Comment (TimWolla) [inline wcfsetup/install/files/lib/system/html/input/node/HtmlInputNodeImg.class.php:277]:
```suggestion
            $style .= "width: {$width}";
```
?

File: wcfsetup/install/files/lib/system/html/input/node/HtmlInputNodeImg.class.php [hunk 2]
```diff
@@ -255,4 +264,28 @@ protected function handleSmiley(\DOMElement $element)
...
+
+        $width = $element->getAttribute("data-width");
+        if ($width && $width !== "100%") {
+            $style = $element->getAttribute("style");
+            if ($style !== "") {
+                $style .= "; ";
+            }
+
+            $style .= "width: {$width}";
+            $element->setAttribute("style", $style);
```
Comment (TimWolla) [inline wcfsetup/install/files/lib/system/html/input/node/HtmlInputNodeImg.class.php:286]: Would two consecutive semicolons cause issues?

**Review by TimWolla — COMMENTED** [review]
I don't think I can review this meaningfully. Doesn't look obviously wrong to me.

**Comment by dtdesign** [comment]
Thanks, but apart from obvious errors there is not a lot to look for. I have tested the changes during development and a lot of it is workaround the quirks of HTMLPurifier and extending the totally not already overloaded `attach` BBCode.

**Pull request merged**